File: ympa_fsharp_giraffe_server/src/model/AddHiddenOffersRequest.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.HiddenOfferDTO

module AddHiddenOffersRequest =

  //#region AddHiddenOffersRequest


  type AddHiddenOffersRequest = {
    HiddenOffers : HiddenOfferDTO[];
  }
  //#endregion

File: src/main/scala/org/openapitools/server/model/AddHiddenOffersRequest.scala
package org.openapitools.server.model


/**
 * Запрос на скрытие оферов.
 *
 * @param hiddenOffers Список скрытых товаров.  for example: ''null''
*/
final case class AddHiddenOffersRequest (
  hiddenOffers: Seq[HiddenOfferDTO]
)

